### PR TITLE
make the bin commands work for any directory name

### PR DESCRIPTION
**Pull request by mwnicks**
### What

Fix bin commands to make them work in all directory names

### CODE REVIEW

File: bin/container [hunk 1]
```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-docker exec -ti phe-cv19-site_web_1 bash
+docker exec -ti ${PWD##*/}_web_1 bash
```
Comment (VivianAllen) [inline bin/container:3]: what does the `##*/` bit do?

Comment (VivianAllen) [reply bin/container:3]: ah trims the results of PWD. thats interesting!

**Review by VivianAllen — APPROVED** [review]
lgtm ✅

File: bin/container (hunk 1 above)
Comment (mwnicks) [reply bin/container:3]: yeah, just returns the last directory in the path

**Pull request merged**